src/translations/pt/core.js

```diff
@@ -14,17 +14,17 @@ export default {
   
   exportTitle: "Compartilhe",
   exportSubtitle:
-    "Os sistemas de design são melhores construídos em conjunto. É por isso que tornamos possível compartilhar este checklist, incluindo os itens que você já verificou. Compartilhe seu progresso com sua equipe para que todos possam acompanhar juntos.",
+    "Os Design Systems são melhores construídos em conjunto. É por isso que tornamos possível compartilhar este checklist, incluindo os itens que você já verificou. Compartilhe seu progresso com sua equipe para que todos possam acompanhar juntos.",
   exportAction: "Compartilhe seu progresso",
   exportComplete: "Link copiado",
   
   moreTitle: "Procurando por mais?",
   moreText:
-    "Usamos este checklist para construir o Reshaped, um sistema de design profissionalmente elaborado para Figma e React. <br /><br /> Ao longo dos anos, vimos tendências surgirem e desaparecerem, enquanto os desafios fundamentais permaneceram. Por isso, escolhemos nos concentrar em coisas que serão relevantes daqui a anos e criamos nossa própria abordagem que funciona em qualquer escala.<br /><br /> Nossa experiência de mais de 10 anos se transformou em um sistema de design white-label que usamos nós mesmos e agora disponibilizamos para outras pessoas.",
+    "Usamos este checklist para construir o Reshaped, um Design System profissionalmente elaborado para Figma e React. Ao longo dos anos, vimos tendências surgirem e desaparecerem, enquanto os desafios fundamentais permaneceram. Por isso, escolhemos nos concentrar em coisas que serão relevantes daqui a anos e criamos nossa própria abordagem que funciona em qualquer escala. Nossa experiência de mais de 10 anos se transformou em um Design System white-label que usamos nós mesmos e agora disponibilizamos para outras pessoas.",
   moreAction: "Saiba mais",
   
   footerTitle: "Precisa de mais do que um checklist?",
   footerText:
-    "Usamos este checklist para construir o Reshaped, um sistema de design profissionalmente elaborado para Figma e React. O Reshaped oferece licenças regulares com atualizações ilimitadas, acesso ao código-fonte do sistema e consultoria focada em sistemas de design.",
+    "Usamos este checklist para construir o Reshaped, um Design System profissionalmente elaborado para Figma e React. Ao longo dos anos, vimos tendências surgirem e desaparecerem, enquanto os desafios fundamentais permaneceram. Por isso, escolhemos nos concentrar em coisas que serão relevantes daqui a anos e criamos nossa própria abordagem que funciona em qualquer escala. Nossa experiência de mais de 10 anos se transformou em um Design System white-label que usamos nós mesmos e agora disponibilizamos para outras pessoas.",
```

Comment:
It seems like this piece adds more content than in the English version of the website and that makes footer much bigger compared to the original. Do you think we can keep it closer to the original here?

Reply:
Yep, makes sense! Not sure why I ended up with such a long paragraph, probably got mixed up with the about section? But I found the correct text in my files, will commit the changes in a bit :)

Reply:
Done!